packages/wmr/test/acorn-traverse.test.js

```diff
@@ -278,6 +278,11 @@ describe('acorn-traverse', () => {
 			expect(str).toMatchInlineSnapshot('"<a.b.c/>;"');
 		});
 
+		it('should serialize JSXNamespacedName (namespaced attributes)', () => {
+			const str = generate(parse(`<x:y a:b="c" />`)).trim();
+			expect(str).toMatchInlineSnapshot('"<x:y a:b="c"/>;"');
```

Comment:
The snapshot doesn't match because those quotes need to be escaped

```suggestion
			expect(str).toMatchInlineSnapshot('"<x:y a:b=\"c\"/>;"');
```